gcloud/bigtable/client.py

```diff
@@ -86,12 +92,22 @@ class Client(_ClientFactoryMixin, _ClientProjectMixin):
                   interact with the Cluster Admin or Table Admin APIs. This
                   requires the :const:`ADMIN_SCOPE`. Defaults to :data:`False`.
 
+    :type user_agent: str
+    :param user_agent: (Optional) The user agent to be used with API request.
+                       Defaults to :const:`DEFAULT_USER_AGENT`.
+
+    :type timeout_seconds: int
+    :param timeout_seconds: Number of seconds for request time-out. If not
+                            passed, defaults to
+                            :const:`DEFAULT_TIMEOUT_SECONDS`.
+
```

Comment:
Then the stubs are called with the specific RPC methods:
https://github.com/dhermes/gcloud-python-bigtable/blob/21aa4a73df0fa7529285271e9fe2cc28b1e992e7/gcloud_bigtable/client.py#L607-L608

But we set the stubs to a permanently entered state (they are context managers):
https://github.com/dhermes/gcloud-python-bigtable/blob/21aa4a73df0fa7529285271e9fe2cc28b1e992e7/gcloud_bigtable/client.py#L503-L521